ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -268,6 +268,482 @@
         "type": "io.confluent.ksql.util.KsqlStatementException",
         "message": "Invalid join condition: foreign-key table-table joins are not supported as part of n-way joins. Got LEFT_TABLE.F1 = RIGHT_TABLE.ID3."
       }
+    },
+    {
+      "name": "value column - inner join",
```

Comment:
What does "value column" mean? Maybe just `Should compute inner join`

Reply:
It means we're joining on a value column from the left source, rather than a key column. See the tests titled "partial key column" for comparison.